.github/renovate.json5

```diff
@@ -71,4 +72,7 @@
   // concurrent PRs limit.
   prConcurrentLimit: 50,
   prHourlyLimit: 5,
+
+  // add renovatebot label to easily filter PRs/email
+  labels: ["renovatebot"],
```

Comment:
nit: maybe `renovate-polaris` to allow filtering renovate emails coming from different projects :)

Reply:
As we don't have consensus here, I removed this part and just focus on label.